base/include/NA6PRecoParam.h

```diff
@@ -0,0 +1,30 @@
+// NA6PCCopyright
```

Comment:
Should not this go to the rec directory? 
Also, since it is specifically for the CA, it would be better to reflect this in the name, in case later we have other reconstructors.

Reply:
About the CA: actually I thought that this class will contain also the parameters of the clusterizer, the vertexer, of other trackers etc. This was just the first core and for the time being it contains only the parameters of the tracker CA, as a starting point. If you think it is better to split the files by "step" of the reconstruction I'll rename it.

Reply:
Concerning the recoparam: I just wanted to avoid it becoming too bulky if we start adding different unrelated options in a single class. But ok, we can split it later if this become a problem.

For the directories: to avoid circular references, the base contains the stuff which both  sim and rec may depend on, the rec can depend on sim